docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -34,6 +34,8 @@ options][link-config].
 | `REDIS_TLS`                          | If `true`, then the connection to the Redis server is protected by TLS authentication. Defaults to `false`                                                                                         | `true`, `false`                                           |
 | `CUBEJS_REDIS_POOL_MAX`              | The maximum number of connections to keep active in the Redis connection pool. Must be higher than `CUBEJS_REDIS_POOL_MIN`. Defaults to `1000`                                                     | A valid number of connections.                            |
 | `CUBEJS_REDIS_POOL_MIN`              | The minimum number of connections to keep active in the Redis connection pool. Must be lower than `CUBEJS_REDIS_POOL_MAX`. Defaults to `2`                                                         | A valid number of connections                             |
+| `CUBEJS_REDIS_USE_IOREDIS`           | Makes the redis connection use ioredis instead of redis npm package. This flag must be true in order for connections to Redis Sentinel to work as well as connections directly to Redis. Defaults to `false`                                                         | `true`, `false`                             |
```

Comment:
```suggestion
| `CUBEJS_REDIS_USE_IOREDIS`           | Use [`ioredis`][gh-ioredis] instead of[ `redis`][gh-node-redis]. This flag must be true in order for connections to Redis Sentinel to work as well as connections directly to Redis. Defaults to `false` | `true`, `false`                             		|
```

Reply:
I've added inline links to the `ioredis` and `redis` repositories, the URL referenced is in the next suggestion